lib/osf-components/addon/components/node-metadata-form/styles.scss

```diff
@@ -11,9 +11,10 @@
         color: $color-link-black;
     }
 
-    div:not(.resource-edit) {
+    div:not(.resource-edit),
+    div:not(.metadata-description-field) {
         justify-content: space-between;
-        margin: 5px 0;
+        margin: 5px 0 15px;
```

Comment:
> Which line would you like changed and how?

This line, try changing margin to padding, then take out the L15 and L159-163.

Reply:
The same here.